docs/tutorials/send-payjoin-with-pdk.md

```diff
@@ -0,0 +1,403 @@
+    );
+}
+
+fn main() {
+    setup_wallets();
+}
+  ```
+
+   And then:
+
+   ```
+   cargo run
+   ```
+
+   And you should see messages for creating the
+   wallets and their balances.
+
+   ```shell
+   [src/main.rs:13] &wallets = [
+       "sender",
+       "receiver",
+   ]
+   [src/main.rs:75] "sender balance: {}" = "sender balance: {}"
+   [src/main.rs:75]
+   sender_client.get_balances().unwrap().mine.trusted.to_btc() =
+   5050.0
+   [src/main.rs:79] "receiver balance: {}" = "receiver
+   balance: {}"
+   [src/main.rs:79]
+   receiver_client.get_balances().unwrap().mine.trusted.to_btc()
+   = 50.0
+   ```
+
+# 4. Sender
+
+  Payjoin sender have two responsibilties:
+
+  1. Get receiver URI, construct an http request with PSBT in the
+     body based on the URI, and send the request to the payjoin receiver.
+  2. Wait for a response from the receiver to confirm the PSBT,
+     finalize and broadcast it.
+
+  The first point is where we for example scan a QR code, with BiP21
+  URI encoded. If the BiP21 has `pj` param then its a valid URI for
+  payjoin and we construct a PSBT based on the bitcoin address and
+  amount(if set) from the URI. After building the request, we send it
+  to the `pj` param value from the URI, which will be an https end
+  point the receiver listening on for incoming requests.
+
+
+  In the following code we add a `sender` struct and also an
+  `https_agent` function. As you would guess, the `https_agent`
+  function is used in order to send the https requests and get the
+  response back whil the `sender` struct and implementation holds the
+  functionality for a payjoin sender.
+
+  In the code below, in the `main` function we also read `args` from
+  the command line to consume the URI and utilise `payjoin::Uri::try_from`
+  to convert the string to a valid URI.
+
+
+  lets add the code to `main.rs` as well, just after the
+  `setup_wallet` function:
+
+  ```rust
+// payjoin-tutorial/src/main.rs
+
+use payjoin::bitcoin::address::NetworkChecked;
+use std::collections::HashMap;
+use std::path::PathBuf;
+use std::str::FromStr;
+use std::sync::Arc;
+
+use bitcoincore_rpc::bitcoin::psbt::Psbt;
+use bitcoincore_rpc::bitcoin::Txid;
+use payjoin::base64;
+use payjoin::send::RequestBuilder;
+use rustls::client::ClientConfig;
+use rustls::{Certificate, RootCertStore};
+  
+const LOCAL_CERT_FILE: &str = "localhost.der";
+
+fn https_agent() -> ureq::Agent {
+    use ureq::AgentBuilder;
+
+    let mut local_cert_path = PathBuf::from("/tmp");
+    local_cert_path.push(LOCAL_CERT_FILE);
+    let cert_der = std::fs::read(local_cert_path).unwrap();
+    let mut root_cert_store = RootCertStore::empty();
+    root_cert_store.add(&Certificate(cert_der)).unwrap();
+    let client_config = ClientConfig::builder()
+        .with_safe_defaults()
+        .with_root_certificates(root_cert_store)
+        .with_no_client_auth();
+
+    AgentBuilder::new()
+        .tls_config(Arc::new(client_config))
+        .build()
+}
```

Comment:
Payjoin Dev Kit uses ureq in order to support msrv, but it's a less familiar crate than `reqwest`. One should be able to use a `reqwest::blocking::Client` with [`danger_accept_invalid_certs`](https://docs.rs/reqwest/latest/reqwest/struct.ClientBuilder.html#method.danger_accept_invalid_certs) to demonstrate this example without reading from a local cert in a file every run